xDripG5/Messages/SessionStartTxMessage.swift

```diff
@@ -12,11 +12,16 @@ import Foundation
 struct SessionStartTxMessage: RespondableMessage {
     typealias Response = SessionStartRxMessage
 
-    let startTime: UInt32
+    /// Time since activation in Dex seconds
+    let time: UInt32
```

Comment:
Why rename this from `startTime`?

Reply:
Just for consistency between CalibrateTx, StartSessionTx and StopSessionTx, and `startTime` and `stopTime` and `calibrateTime` seem to me a bit verbose. Happy to revert to `startTime` and `stopTime`. Reverted.